istioctl/cmd/proxystatus.go

```diff
@@ -103,14 +109,26 @@ Retrieves last sent and last acknowledged xDS sync from Istiod to each Envoy in
 			if err != nil {
 				return err
 			}
+
 			sw := pilot.StatusWriter{Writer: c.OutOrStdout()}
+
+			switch statusArgs.output {
+			case jsonFormat:
+				return sw.PrintJSON(statuses)
+			case tableFormat:
+			default:
+				return fmt.Errorf("unknown format: %s", statusArgs.output)
+			}
+
 			return sw.PrintAll(statuses)
 		},
 	}
 
 	opts.AttachControlPlaneFlags(statusCmd)
 	statusCmd.PersistentFlags().StringVarP(&configDumpFile, "file", "f", "",
 		"Envoy config dump JSON file")
+	statusCmd.PersistentFlags().StringVarP(&statusArgs.output, "output", "o", tableFormat, "Output format for revision description "+
```

Comment:
The description should be related to `proxy status` not `revision`.

Reply:
Good catch thanks